apps/coscrad-frontend/src/components/header/header.tsx

```diff
@@ -17,6 +19,7 @@ const StyledAppBar = styled(AppBarWithTestIdDataAttribute)({
 const StyledToolbar = styled(Toolbar)({
     display: 'flex',
     justifyContent: 'space-between',
+    height: '64px',
```

Comment:
This `64px` seems to scrunch the text in the app bar on the phone.  This is why I find CSS so annoying because I fix one thing and then something else gets messed up.  If this 64 is needed, we could just work on the font size.

<img width="332" alt="image" src="https://github.com/COSCRAD/coscrad/assets/63014783/0f555e73-2dfd-4eae-960e-5088b0ddd7d0">